modules/apps/object/object-test/src/testFunctional/tests/ObjectFieldsDateAndTime.testcase

```diff
@@ -0,0 +1,68 @@
+@component-name = "portal-object"
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Object";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+	}
+
+	tearDown {
+		ObjectAdmin.deleteAllCustomObjectsViaAPI();
+
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if (${testPortalInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@description = "LPS-143068 - Verify if is possible to add DateTime on entries from Custom Object"
+	@priority = 5
+	test CanAddEntry {
+		property portal.acceptance = "true";
+
+		task ("Given a custom object with a date and time field as input as entered") {
```

Comment:
Given a custom object with a date and time field and the time storage set to user input